org/allPRs.ts

```diff
@@ -1,4 +1,4 @@
-import { schedule, danger, warn, fail } from "danger"
+import { schedule, danger, warn, fail, peril, GitHubUser } from "danger"
```

Comment:
I don't see these two new imports used elsewhere in the file. Still needed?

Reply:
looks like not; removed!